lib/Notes/components/NotesAccordion/components/NotesList/NotesList.js

```diff
@@ -51,13 +51,16 @@ class NotesList extends React.Component {
     },
   }
 
-  state = { isDetailsExpanded: false };
+  state = { isDetailsExpanded: {} };
 
-  handleShowMoreClick = (e) => {
+  handleShowMoreClick = ({ id, e }) => {
```

Comment:
Please, use currying to be able to bind the `id` to the function:
`handleShowMoreClick = id => e => {`